tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -996,7 +997,9 @@ def _wait_for_partition_state_changed(scheduler_commands, partition, desired_sta
 def _update_and_start_cluster(cluster, config_file):
     cluster.stop()
     _wait_for_computefleet_changed(cluster, "STOPPED")
+    time.sleep(150)
     cluster.update(str(config_file), force_update="true")
+    time.sleep(150)
```

Comment:
please add a comment in the code explaining why this timeout is required.

Reply:
Added